bundles/org.openhab.binding.qbus/src/main/java/org/openhab/binding/qbus/internal/protocol/QbusCommunication.java

```diff
@@ -121,11 +121,25 @@ public synchronized void startCommunication() throws IOException, InterruptedExc
         InetAddress addr = InetAddress.getByName(handler.getAddress());
         Integer port = handler.getPort();
 
+        Socket socket = null;
+
         if (port != null) {
-            Socket socket = new Socket(addr, port);
-            qSocket = socket;
-            qOut = new PrintWriter(socket.getOutputStream(), true);
-            qIn = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            try {
+                socket = new Socket(addr, port);
+                qSocket = socket;
+                qOut = new PrintWriter(socket.getOutputStream(), true);
+                qIn = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            } catch (IOException e) {
+
+            } finally {
+                if (socket == null) {
+                    Integer serverCheck = handler.getServerCheck();
+                    handler.bridgeOffline(ThingStatusDetail.COMMUNICATION_ERROR,
+                            "No communication with Qbus Server, will try to reconnect every " + serverCheck
+                                    + " minutes");
+                    return;
```

Comment:
Are you sure this code is working? The code in `finally` will also be executed if no exception raised.

Please handle the exception appropriately. E.g. set the Thing offline.